tests/amp-tag-and-attribute-sanitizer-private-methods-tests.php

```diff
@@ -713,6 +713,13 @@ public function get_replace_node_with_children_data() {
 				),
 				'',
 			),
+			'nested_invalid_elements' => array(
+				array(
+					'source' => '<div><details><summary><p>Example Summary</p></summary><p>Example expanded text</p></details></div>',
```

Comment:
Now that AMP allows the `details` element (via https://github.com/ampproject/amphtml/pull/18440), the tests here aren't relevant now.